ibft/sync/history.go

```diff
@@ -100,36 +100,9 @@ func (s *HistorySync) findHighestInstance() (*proto.SignedMessage, string, error
 		usedPeers = usedPeers[:4]
 	}
 
-	// fetch response
```

Comment:
@amir-blox maybe we should expand the function description to also include expected behaviour